packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -670,12 +670,12 @@ class RenderProperties {
     Size realSize,
     this.layoutProperties,
     this.isFreeSpace = false,
-  })  : width = size?.width,
-        height = size?.height,
-        realWidth = realSize?.width,
-        realHeight = realSize?.height,
-        dx = offset?.dx,
-        dy = offset?.dy;
+  })  : width = size?.width ?? 0.0,
+        height = size?.height ?? 0.0,
+        realWidth = realSize?.width ?? 0.0,
+        realHeight = realSize?.height ?? 0.0,
+        dx = offset?.dx ?? 0,
+        dy = offset?.dy ?? 0;
```

Comment:
do these need to be explicit doubles (e.g. 0.0)?

Reply:
they don't but they should look consistent with the other cases which used 0.0.
done.